client/landing/stepper/declarative-flow/tailored-ecommerce-flow.ts

```diff
@@ -60,11 +60,16 @@ export const ecommerceFlow: Flow = {
 		const locale = useLocale();
 		const siteSlugParam = useSiteSlugParam();
 		const site = useSite();
+		const flags = new URLSearchParams( window.location.search ).get( 'flags' );
 
 		const getStartUrl = () => {
```

Comment:
I worry this is getting a little hard to read. What if we did something like this instead?

```
const getStartUrl = () => {
	let url =
		locale && locale !== "en"
			? `/start/account/user/${locale}?variationName=${flowName}&redirect_to=/setup/ecommerce/storeProfiler`
			: `/start/account/user?variationName=${flowName}&redirect_to=/setup/ecommerce/storeProfiler`;

	return url + (flags && `?flags=${flags}`);
};
```

Otherwise, it worked as expected!